src/pages/ReimbursementAccount/BusinessInfo/substeps/WebsiteBusiness.tsx

```diff
@@ -0,0 +1,99 @@
+import Str from 'expensify-common/lib/str';
+import React, {useEffect, useMemo} from 'react';
+import type {OnyxEntry} from 'react-native-onyx';
+import {withOnyx} from 'react-native-onyx';
+import FormProvider from '@components/Form/FormProvider';
+import InputWrapper from '@components/Form/InputWrapper';
+import Text from '@components/Text';
+import TextInput from '@components/TextInput';
+import useLocalize from '@hooks/useLocalize';
+import type {SubStepProps} from '@hooks/useSubStep/types';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as ValidationUtils from '@libs/ValidationUtils';
+import * as BankAccounts from '@userActions/BankAccounts';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type {ReimbursementAccount, Session, User} from '@src/types/onyx';
+import type {FormValues} from '@src/types/onyx/Form';
+import type * as OnyxCommon from '@src/types/onyx/OnyxCommon';
+
+type WebsiteBusinessOnyxProps = {
+    /** Reimbursement account from ONYX */
+    reimbursementAccount: OnyxEntry<ReimbursementAccount>;
+
+    /** Session info for the currently logged in user. */
+    session: OnyxEntry<Session>;
+
+    /** Object with various information about the user */
+    user: OnyxEntry<User>;
+};
+
+type WebsiteBusinessProps = WebsiteBusinessOnyxProps & SubStepProps;
+
+const companyWebsiteKey = CONST.BANK_ACCOUNT.BUSINESS_INFO_STEP.INPUT_KEY.COMPANY_WEBSITE;
+
+const validate = (values: FormValues): OnyxCommon.Errors => {
```

Comment:
NAB: It is a bit inconsistent, sometimes validate function is defined in the component body and sometimes outside

Reply:
It's because in most cases we don't need to keep it inside the component because the function uses only the param. The validation function is defined inside the component in cases when we need e.g prop.